pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -155,6 +155,11 @@ type GrpcPodConfig struct {
 	//
 	// This field is best-effort; if unset, no default will be used and no Pod memory limit or $GOMEMLIMIT value will be set.
 	MemoryTarget *resource.Quantity `json:"memoryTarget,omitempty"`
+
+	// UseWellKnownServer configures the gRPC catalog Pod to use a well-known version of the `opm` server. The catalog
+	// index image that this CatalogSource is configured to use *must* be using the file-based catalogs in order to
+	// utilize this feature.
+	UseWellKnownServer bool `json:"useWellKnownServer,omitempty"`
```

Comment:
What made you move away from the proposed API in https://github.com/operator-framework/operator-lifecycle-manager/issues/3019?

```yaml
    extractContent:
      cacheDir: /var/cache/
      configDir: /var/config/
```